### PR TITLE
White corpsman helmet

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Adds white corpsman helmet, which is mostly insprited by Aliens Dark Descent analogue and historical counterparts

<details>

image

image

</details>

Has identical stats, just a reskin.

Can be only obtained in corpsman and synth snowflake vendor.

Sprites by Crowford, discord @sir.crow.

# Explain why it's good for the game

More lore friendly customization - good.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

dreamseeker_QYcvBP3544

4NW2tY3jF7

</details>

# Changelog
:cl:
add: white corpsman helmet
/:cl:

### CODE REVIEW

**Comment by hislittlecuzingames** [comment]
guess I'll get capped more than normal. 

👍

**Comment by Git-Nivrak** [comment]
Speedmerge

**Comment by s5nt** [comment]
good pr but its pretty much a "TARGET ME!" beacon

**Comment by Steelpoint** [comment]
Corpsman will simply have a choice between making themselves very visible or fairly hidden to a xenomorph, this isn't a nerf since the choice remains optional and non-corpsman helmets are freely issued at squad reqs.

File: code/game/machinery/vending/vendor_types/squad_prep/squad_medic.dm [hunk 1]
```diff
@@ -137,6 +137,10 @@ GLOBAL_LIST_INIT(cm_vending_clothing_medic, list(
 		list("Medium Armor", 0, /obj/item/clothing/suit/storage/marine/medium, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_RECOMMENDED),
 		list("Heavy Armor", 0, /obj/item/clothing/suit/storage/marine/heavy, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_REGULAR),
 
+		list("HELMET (CHOOSE 1)", 0, null, null, null),
+		list("M10 corpsman helmet", 0, /obj/item/clothing/head/helmet/marine/medic, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_REGULAR),
+		list("M10 white corpsman helmet", 0, /obj/item/clothing/head/helmet/marine/medic/white, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_REGULAR),
```
Comment (Zonespace27) [inline code/game/machinery/vending/vendor_types/squad_prep/squad_medic.dm:142]: Capitalize

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Review by Zonespace27 — APPROVED** [review]
Code seems fine

**Comment by Nomoresolvalou** [comment]
I think you should also add this to the synth snowflake vendor, since they also have the normal corpsman helmet there as well.

**Comment by Blundir** [comment]
> I think you should also add this to the synth snowflake vendor, since they also have the normal corpsman helmet there as well.

Done

**Review by harryob — APPROVED** [review]

**Pull request merged**